app/src/main/kotlin/org/koitharu/kotatsu/image/ui/ImageActivity.kt

```diff
@@ -98,22 +98,14 @@ class ImageActivity : BaseActivity<ActivityImageBinding>(), ImageRequest.Listene
 			.enqueueWith(coil)
 	}
 
-	private class SsivTarget(
+	private data class SsivTarget(
 		override val view: SubsamplingScaleImageView,
 	) : ViewTarget<SubsamplingScaleImageView> {
 
 		override fun onError(error: Drawable?) = setDrawable(error)
 
 		override fun onSuccess(result: Drawable) = setDrawable(result)
 
-		override fun equals(other: Any?): Boolean {
-			return (this === other) || (other is SsivTarget && view == other.view)
```

Comment:
This one is not the same as a generated `equals` method, so, this class should not be a `data` class